### PR TITLE
Lesson 18

### PR DESCRIPTION
**Pull request by carlotrimarchi**

### CODE REVIEW

**Review by haraldF — APPROVED** [review]

File: lesson18.md [hunk 1]
```diff
@@ -0,0 +1,385 @@
...
+---
+
+### Refresher: Data Types
+
+```js
+// string
+let name = "Owen";
+
+// number
+let age = 28;
```
Comment (owenlow) [inline lesson18.md]: 😛
```suggestion
let age = 29;
```

File: lesson18.md [hunk 2]
```diff
@@ -0,0 +1,385 @@
...
+### Bonus Exercise
+
+* Create a new array containing only numbers of the original array that are smaller than 5
+ * ex: given input `[1, 3, 7, 2, 9, 5]`, return `[1, 3, 2]`
+* Write a function that takes a number and an array of numbers as input parameter. Return the index of the number if it is in the array, otherwise return `-1`
+ * ex: given inputs `5` and `[2, 1, 5, 6, 3]`, return `2`
+
+---
+
+### Solution
```
Comment (owenlow) [inline lesson18.md]: Have we had any feedback on solutions in the slides? My concern would be that it makes it too easy for them to just look ahead to get the solutions.

File: lesson18.md [hunk 3]
```diff
@@ -0,0 +1,385 @@
...
+* Output the largest number of the array
+* Output the average (mean) of all numbers (sum of all numbers divided by amount of numbers)
+
+Create an array containing a few numbers, ex `[3, 2, 9, 5, 6]` and test your code:
+
+```js
+function exercise3(array) {
+    // your code here
+}
+exercise3([3, 2, 9, 5, 6]);
```
Comment (owenlow) [inline lesson18.md]: IMO, when we teach new syntax we should keep things as simple as possible. Some students have difficulty understanding inlined arguments:

```suggestion
let myNumbers = [3, 2, 9, 5, 6];
exercise3(myNumbers);
```

**Review by owenlow — APPROVED** [review]
some nitpicks, looks good to me!

File: lesson18.md (hunk 1 above)
Comment (carlotrimarchi) [reply lesson18.md]: maybe @haraldF  can update his script so that it computes the correct age at every semester

File: lesson18.md (hunk 2 above)
Comment (carlotrimarchi) [reply lesson18.md]: yes, I was thinking the same. I'll remove them tomorrow

File: lesson18.md (hunk 3 above)
Comment (carlotrimarchi) [reply lesson18.md]: yes, completely agree, totally overlooked this

File: lesson18.md [hunk 4]
```diff
@@ -0,0 +1,385 @@
...
+ * ex: given input `[1, 3, 7, 2, 9, 5]`, return `[1, 3, 2]`
+* Write a function that takes a number and an array of numbers as input parameter. Return the index of the number if it is in the array, otherwise return `-1`
+ * ex: given inputs `5` and `[2, 1, 5, 6, 3]`, return `2`
+
+---
+
+### Solution
+
+```js
+function exercise3(array) {
```
Comment (sevtapim) [inline lesson18.md:361]: I would split up each exercise into its own separate function for clarity.

**Review by sevtapim — APPROVED** [review]

**Pull request merged**